pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -80,7 +104,19 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			continue
 		}
 
-		_, err := opc.pvcLister.PersistentVolumeClaims(ns).Get(pvcName)
+		var err error
+		// check informer cache
+		_, err = opc.pvcLister.PersistentVolumeClaims(ns).Get(pvcName)
+		if err == nil {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPVCIsFound
+			continue
+		}
+		if !errors.IsNotFound(err) {
+			return skipReason, err
+		}
+
+		// if PVC not found in cache, re-check from apiserver directly to make sure the PVC really not exist
+		_, err = opc.kubeCli.CoreV1().PersistentVolumeClaims(ns).Get(pvcName, metav1.GetOptions{})
```

Comment:
Should the default QPS and Burst be increased for the kubeclient?

Reply:
we can see perf result in stability test and production, if it has a problem (operates too slow because QPS limit), we should try to optimize our code to avoid unnecessary API requests before increasing client-go QPS/Burst